docs/docsite/rst/dev_guide/developing_collections_testing.rst

```diff
@@ -14,7 +14,7 @@ Testing tools
 =============
 
 The main tool for testing collections is ``ansible-test``, Ansible's testing tool described in :ref:`developing_testing` and provided by both the ``ansible`` and ``ansible-core`` packages.
-
+Use ansible-test tool from your collection directory, which must include ansible_collections in the path, for example collections/ansible_collections/community.general for the community.general collection. See :ref:testing_collections_guide for more details.
```

Comment:
```suggestion
Use ansible-test tool from your collection directory, which must include ansible_collections in the path, for example collections/ansible_collections/community/general for the community.general collection. See :ref:testing_collections_guide for more details.
```

The path should have a `/` instead of a `.` between the namespace and collection name.